src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+class Deadline extends Task {
+    private String dueTime;
+
+    /**
+     * Constructor
+     * 
+     * @param name task name
+     * @param dueTime task due date/time as a String
+     * @return new task instance
+     */
+    public Deadline(String name, String dueTime) {
+        super(name);
+        this.dueTime = dueTime;
+    }
+
+    @Override
+    public String toString() {
+        String s = "[D]" + super.toString();
+        return s + " (due by " + this.dueTime + ")";
+    }
+}
```

Comment:
For the deadline input, consider adding in the functionality to take in date time inputs.